steps/deploy-huawei-app-gallery/1.0.0/step.yml

```diff
@@ -0,0 +1,66 @@
+title: Deploy to Huawei App Gallery
+summary: |
+  Uploads APK to Huawei App Gallery
+description: |
+  Uploads APK to Huawei App Gallery
+website: https://github.com/FutureMind/bitrise-step-huawei-app-gallery-apk-distribution
+source_code_url: https://github.com/FutureMind/bitrise-step-huawei-app-gallery-apk-distribution
+support_url: https://github.com/FutureMind/bitrise-step-huawei-app-gallery-apk-distribution/issues
+published_at: 2020-01-08T16:58:18.663184+01:00
+source:
+  git: https://github.com/FutureMind/bitrise-step-huawei-app-gallery-apk-distribution.git
+  commit: fe29d259c54362ac4db1fbb7cf8f2a4e770c2ee6
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  - name: jq
+  apt_get:
+  - name: git
+  - name: wget
+  - name: jq
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- apk_path: null
```

Comment:
It should probably default to `$BITRISE_APK_PATH`

Reply:
Added BITRISE_APK_PATH as default. AAB is not supported by App Gallery.